site/scripts/lintChangesets.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-console */
+import { glob, readFile } from 'fs/promises';
+import { resolve } from 'path';
+import { styleText } from 'util';
+
+import { supportedLanguages } from '../src/App/Code/supportedLanguages.ts';
+
+const braidChangesetRegex =
+  /^---\n'braid-design-system': (?:patch|minor|major)\n---$/m;
+
+const markdownCodeBlockLanguageRegex = /``` *(?<language>[a-z]*)\n/g;
```

Comment:
What are your thoughts on tweaking this to only find code blocks with a language?

```suggestion
const markdownCodeBlockLanguageRegex = /```(?<language>[a-z]+)\n/g;
```